frames/src/main/java/com/checkout/frames/component/base/InputComponent.kt

```diff
@@ -6,26 +6,6 @@ import androidx.compose.foundation.layout.wrapContentHeight
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.tooling.preview.Preview
```

Comment:
Are you also deleting old code as we go ?

Reply:
It was a preview for InputComponent, for debug/testing.